kronolith/lib/Application.php

```diff
@@ -988,11 +988,11 @@ public function davGetObjects($collection)
         // Handle external collections
         $exploded = explode(':', $internal, 2);
         $driverType = null;
-        if ($exploded[0] == "external") {
-            $driverType = "Horde";
+        if ($exploded[0] == 'external') {
+            $driverType = 'Horde';
             $internalName = str_replace(':', '/', $exploded[1]);
         } elseif (!Kronolith::hasPermission($internal, Horde_Perms::SHOW)) {
-            throw new Kronolith_Exception(_("Calendar does not exist or no permission to edit"));
+            throw new Kronolith_Exception(_('Calendar does not exist or no permission to edit'));
```

Comment:
Done. I was not aware.

CODING_STANDARDS shows the double quote example in section 23 internatl. , but does not instruct to use double quotes. In 27 quotes it says **Should** be used in the gettext shortcut _("") format.